sources/packages/api/src/database/migrations/1626994719103-UpdateApplicationsAddAssessment.ts

```diff
@@ -0,0 +1,18 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+import { getSQLFileData } from "../../utilities";
+
```

Comment:
The formatter doesn't seem to be working, please check and format the file accordingly.
Please check on others files on API also.